pkg/registry/core/service/storage/storage_test.go

```diff
@@ -5593,6 +5593,64 @@ func TestCreateDryRun(t *testing.T) {
 	}
 }
 
+func TestDeleteTypes(t *testing.T) {
+	testCases := []struct {
+		name string
+		svc  *api.Service
+	}{{
+		name: "type:ExternalName",
+		svc: svctest.MakeService("foo",
+			svctest.SetTypeExternalName),
+	}, {
+		name: "type:ClusterIP",
```

Comment:
Commit `Service REST test: clean up redundant delete tests`:
Missing duals-tack with two IPs to test allocator didn't not mess up and just freed on ClusterIP

Reply:
This whole test should be removed after the feature tests prove create-update-delete more completely.  I'll leave this as-is here and remove it in the later commit